source/educators/how-tos/proctored_exams/enable_proctored_exams.rst

```diff
@@ -20,13 +20,11 @@ To enable proctored exams in your course, follow these steps.
 
 #. Select **Submit**. You can now create proctored exams in your course.
 
+
 Configuring Proctoring Provider
 *******************************
 
-The choice of proctoring provider can be configured. Since Fall 2019,
-the default proctoring provider is **RPNow** by PSI.  To use
-**Proctortrack** by Verificient, please follow these steps to change
-the proctoring provider:
+The default proctoring provider is **RPNow** by PSI. You can configure the proctoring provider by following these steps:
```

Comment:
Can you still configure a proctoring provider if Proctortrack is no longer available?